inst/markdown/compareScenarios2/cs2_01_summary.Rmd

```diff
@@ -4,7 +4,7 @@
 
 ## GHG Emissions
 
-```{r Emi-GHG}
```

Comment:
Did the linter complain about this? Or did you just remove because it is kinda redundant?

Reply:
No, but they are redundant and if you happen to label two sections with the same name, everything crashes. This can easily happen if you split a chunk via shortcut; then both resulting chunks have the same name. So better to not use labels for chunks unless they are actually useful.